File: atcoder/abc405a_is_it_rated.cpp
// vicfred
// https://atcoder.jp/contests/abc405/tasks/abc405_a
// implementation
#include <iostream>

using namespace std;

int main() {
  int r, x;
  cin >> r >> x;
  if(x == 1 and r >= 1600 and r < 3000) {
    cout << "Yes" << endl;
    return 0;
  }
  if(x == 2 and r >= 1200 and r < 2400) {
    cout << "Yes" << endl;
    return 0;
  }
  cout << "No" << endl;
  return 0;
}

File: atcoder/abc405a_is_it_rated.hs
-- uninhm
-- https://atcoder.jp/contests/abc405/tasks/abc405_a
-- implementation

getInts :: IO [Int]
getInts = map read . words <$> getLine

solve :: Int -> Int -> String
solve 1 r
  | 1600 <= r && r <= 2999 = "Yes"
  | otherwise              = "No"
solve 2 r
  | 1200 <= r && r <= 2399 = "Yes"
  | otherwise              = "No"

main :: IO ()
main = do
  [r, x] <- getInts
  putStrLn $ solve x r
